c_tests/test_Reader.cpp

```diff
@@ -563,11 +573,13 @@ TEST_CASE_METHOD(CziCreatorBigM, "test_bigm_czifile", "[Reader_bbox]")
   auto dSizes = czi->dimSizes();
 
   auto ans = czi->getAllSceneYXSize(0, true);
-  assert(ans.size() == 88); // 2 channels * 44 m_index
-  assert(ans[2].x == 22739);
-  assert(ans[2].y == 19201);
-  assert(ans[2].w == 950);
-  assert(ans[2].h == 650);
+  REQUIRE(ans.size() == 88); // 2 channels * 44 m_index
+  REQUIRE(ans[2].x == 23594);
+  REQUIRE(ans[2].y == 19201);
+  REQUIRE(ans[2].w == 950);
+  REQUIRE(ans[2].h == 650);
+  libCZI::CDimCoordinate cdims = {{libCZI::DimensionIndex::C, 0}};
```

Comment:
might as well remove this line and the line below it

Reply:
Should I put it in a conditional #ifndef WIN32? Basically, it's only windows that has a problem with it but it kills the automated build/test process.

Reply:
I'd say either figure out windows issue or remove test.

Reply:
So I removed the line and put a comment in place about the incompatibility so I don't repeat the debugging process. 
At a later point, I'll see if any updates to catch2 make this work on windows.